### DISCUSSION TITLE
Intraday Highs in API

### DISCUSSION THREAD

**Discussion by chrismichener1** [weather-gov/api#602] [General]
I was wondering if there any methods in the API that allows you to see the high temperature so far today for a given station. I believe the NWS quality control checks data before publishing the mid-day climate report and the final daily climate report. I'm looking for something that previews the high that may appear on the daily climate report. Does that ring a bell at all? Thank you!

**Comment by akrherz** [comment]
This does not exist.  The NWS only has access to the 6-hourly METAR high and low reports, which covers a quasi inter-day value, but this value is only disseminated every six hours.  Outside of that value, they would have to manually dial the station to get this information, so it is not routinely done at all.

**Reply by chrismichener1** [reply]
Darn, but thank you!
